pkg/manifests/manifests.go

```diff
@@ -82,6 +83,11 @@ const (
 	hostNotifierDir                 = "/run/rte"
 )
 
+const (
+	latesK8sVersion  = "1.24"
```

Comment:
the problem here is "latest" is "latest at time of writing", which will be obsolete pretty fast. Let's be explicit with versions